### PR TITLE
Editorial suggestions on TLS

### PR DESCRIPTION
**Pull request by MikeBishop**
I believe these are all editorial, but please review.

### CODE REVIEW

File: draft-ietf-quic-tls.md [hunk 1]
```diff
@@ -937,34 +937,36 @@ Note:
...
-protection scheme is defined for the ciphersuite.  This document defines a
-header protection scheme for all ciphersuites defined in {{!TLS13}} aside from
-TLS_AES_128_CCM_8_SHA256.  These ciphersuites have a 16-byte authentication tag
-and produce an output 16 bytes larger than their input.
+All QUIC packets other than Version Negotiation are protected with an AEAD
+algorithm ({{!AEAD}}). Prior to establishing a shared secret, packets are
+protected with AEAD_AES_128_GCM.  Retry packets use the AEAD for integrity
+protection only, as described in {{retry-integrity}}.  Initial packets use a key
+derived from the Destination Connection ID in the client's first Initial packet
+(see {{initial-secrets}}). This provides protection against off-path attackers
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md:954]:
```suggestion
derived from the Destination Connection ID in the client's first Initial packet;
see {{initial-secrets}}. This provides protection against off-path attackers
```

File: draft-ietf-quic-tls.md [hunk 2]
```diff
@@ -973,7 +975,7 @@ order are left-padded with zeros to the size of the IV.  The exclusive OR of the
 padded packet number and the IV forms the AEAD nonce.
 
 The associated data, A, for the AEAD is the contents of the QUIC header,
-starting from the flags byte in either the short or long header, up to and
+starting from the initial byte of either the short or long header, up to and
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md:978]:
```suggestion
starting from the first byte of either the short or long header, up to and
```

"initial" is a little loaded.

File: draft-ietf-quic-tls.md [hunk 3]
```diff
@@ -1085,13 +1087,13 @@ Short Header Packet {
...
-MUST be specified for the AEAD used with that ciphersuite.  This document
-defines algorithms for AEAD_AES_128_GCM, AEAD_AES_128_CCM, AEAD_AES_256_GCM
-(all AES AEADs are defined in {{!AEAD=RFC5116}}), and
-AEAD_CHACHA20_POLY1305 {{!CHACHA=RFC8439}}.  Prior to TLS selecting a
-ciphersuite, AES header protection is used ({{hp-aes}}), matching the
-AEAD_AES_128_GCM packet protection.
+Before a TLS cipher suite can be used with QUIC, a header protection algorithm
+MUST be specified for the AEAD used with that cipher suite.  This document
+defines algorithms for AEAD_AES_128_GCM, AEAD_AES_128_CCM, AEAD_AES_256_GCM (all
+AES AEADs are defined in {{!AEAD=RFC5116}}), and AEAD_CHACHA20_POLY1305 (defined
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md:1093]:
```suggestion
these AES AEADs are defined in {{!AEAD=RFC5116}}), and AEAD_CHACHA20_POLY1305 (defined
```

File: draft-ietf-quic-tls.md [hunk 4]
```diff
@@ -1110,7 +1112,7 @@ sample.
 To ensure that sufficient data is available for sampling, packets are padded so
 that the combined lengths of the encoded packet number and protected payload is
 at least 4 bytes longer than the sample required for header protection.  The
-ciphersuites defined in {{?TLS13}} - other than TLS_AES_128_CCM_8_SHA256, for
+cipher suites defined in {{!TLS13}} - other than TLS_AES_128_CCM_8_SHA256, for
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md:1115]: I tend to use ? where the reference is normative.

If you follow this discipline and remove the last normative reference, the right thing happens.  But I can't see that happening here, so this change is fine.

File: draft-ietf-quic-tls.md [hunk 5]
```diff
@@ -1634,16 +1636,19 @@ messages, that tampering will cause the TLS handshake to fail.
 
 # QUIC-Specific Adjustments to the TLS Handshake
 
-QUIC uses the TLS handshake for more than just negotiation of cryptographic
-parameters.  The TLS handshake provides preliminary values for QUIC transport
-parameters and allows a server to perform return routability checks on clients.
+Because QUIC provides different capabilities than TCP, certain aspects of the
+TLS handshake are different when used with QUIC.
```
Comment (martinthomson) [inline draft-ietf-quic-tls.md:1640]:
```suggestion
Certain aspects of the TLS handshake are different when used with QUIC.
```

File: draft-ietf-quic-tls.md (hunk 2 above)
Comment (MikeBishop) [reply draft-ietf-quic-tls.md:978]: IIRC, I was echoing language from elsewhere, so you might want to see where else you used "initial."

**Pull request merged**